src/constraints/transport.jl

```diff
@@ -131,8 +187,6 @@ function add_transport_constraints!(
             ],
         )
     end
-
-    return
```

Comment:
The formatter was supposed to enforce adding a `return` to the end of functions (we have an option for that). I'm guessing this is another random issue with the formatter version. We should consider alternatives in October.
No action required.